packages/calling/src/CallingClient/calling/call.ts

```diff
@@ -1223,6 +1235,7 @@ export class Call extends Eventing<CallEventTypes> implements ICall {
       this.mediaNegotiationCompleted = false;
       this.sendCallStateMachineEvt({type: 'E_CALL_ESTABLISHED'});
     }
+    this.setTimeoutForState(EXTENDED_CALL_STATE_TIMER);
```

Comment:
I think this might lead to a race condition bug. We're setting the timeout after sending a signal to the state machine right?

Reply:
Using xstate after property to handle timeouts, this condition will not in that